### PR TITLE
Set free tier thresholds in `Constance` configuration

### PR DESCRIPTION
**Pull request by noliveleger**
## Checklist

1. [x] If you've added code that should be tested, add tests
2. [x] If you've changed APIs, update (or create!) the documentation
3. [x] Ensure the tests pass
4. [x] Make sure that your code lints and that you've followed [our coding style](https://github.com/kobotoolbox/kpi/blob/master/CONTRIBUTING.md)
5. [x] Write a description of your work suitable for publishing on [our forum](https://community.kobotoolbox.org/tag/release-notes)
6. [ ] Mention any related issues in this repository (as #ISSUE) and in other repositories (as kobotoolbox/other#ISSUE)
7. [ ] Open an issue in the [docs](https://github.com/kobotoolbox/docs/issues/new) if there are UI/UX changes

## Description

Make free tier thresholds dynamic and expose values in the API (environment endpoint)

### CODE REVIEW

**Review by JacquelineMorrissette — CHANGES_REQUESTED** [review]
Ready to merge once the commented out code is removed

File: kpi/tests/api/test_api_environment.py [hunk 1]
```diff
@@ -49,22 +49,10 @@ def setUp(self):
...
                     ('ar', 'العربيّة'), x
                 ),
-            #'transcription_languages': lambda x: \
-            #    self.assertGreater(len(x), 50) and self.assertIn(
-            #        'uk-UA', x
-            #    ),
-            #'translation_languages': lambda x: \
-            #    self.assertGreater(len(x), 50) and self.assertIn(
-            #        'fa-IR', x
-            #    ),
```
[the remark this answers is not in the document]
Comment (noliveleger) [reply kpi/tests/api/test_api_environment.py:67]: Yep, (AFAIK) those properties are not exposes in this endpoint anymore.

**Comment by bufke** [comment]
See internal [notion doc](https://www.notion.so/kobotoolbox/Configurable-server-limits-5e3d699f11dc4f10b098a869b12d3fd8#e32875d513be4b109c7c86f8b81ad5ad) for default limits. There's 4 limits there.

File: kobo/settings/base.py [hunk 2]
```diff
@@ -310,9 +310,23 @@
         30,
         "Number of days to keep asset snapshots"
     ),
+    'FREE_TIER_THRESHOLDS': (
+        json.dumps({
+            'storage': int(5 * 1024 * 1024 * 1024 * 0.9),  # 90 % of 5 GB
```
Comment (bufke) [inline kobo/settings/base.py:315]: Why 90%?

Comment (noliveleger) [reply kobo/settings/base.py:315]: Totally random.

Comment (bufke) [reply kobo/settings/base.py:315]: Let's use a better default. Either our current internal plan or what we think is a better default for self host users looking to cut down on single users taking up system resources. I don't think 90% makes sense. We could later choose to notify users at 90% usage, but I read this setting as the actual limit at which you cannot submit more (not yet implemented).

Comment (noliveleger) [reply kobo/settings/base.py:315]: Agreed. I've changed them to examples in Notion.

**Pull request merged**